test/kube_testing/k8s.go

```diff
@@ -486,6 +486,10 @@ func (l *K8s) CreatePodsRaw(timeout time.Duration, failTest bool, templates ...*
 	return pods, err
 }
 
+func (l *K8s) GetPod(pod *v1.Pod) (*v1.Pod,error) {
```

Comment:
Would it make sense to just have 2 arguments `name`,`namespace`. The functions looks a bit awkward now.